app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_name(command: str) -> None:
+    parts = command.split(" ")
```

Comment:
You should check if the command starts with 'cp' and contains exactly three parts. This is required to ensure the command is valid as per the task description.